exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -182,3 +182,127 @@ processors:
+This will delete all existing data for the metric, but will allow it to be
+written as a double going forward. The simplest way to do this is by using the
+"Try this method" tab in the API reference for
+[DeleteMetricDescriptor](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/delete)
+
+### Points Written Too Frequently
+
+Error: `One or more points were written more frequently than the maximum sampling period configured for the metric.`
+
+Google Managed Service for Promethueus (and Google Cloud Monitoring)
+[limit](https://cloud.google.com/monitoring/quotas#custom_metrics_quotas) the
+rate at which points can be written to one point ever 5 seconds. If you try to
+write points more frequently, you will encounter the error above. If you know
+that aren't writing points more frequently than 5 seconds, this can be a
+symptom of the Timeseries Collision problem below.
+
+### Timeseries Collision
+
+Error: `Duplicate TimeSeries encountered. Only one point can be written per TimeSeries per request.`
+
+Error: `Points must be written in order. One or more of the points specified had an older start time than the most recent point.`
+
+#### Explanation for Errors
+
+The errors above, and sometimes the
+`points were written more frequently than the maximum sampling period` error
+can indicate that two metric datapoints are being written without any resource
+or metric attributes that distinguish them from each other. We refer to this as
+a "Timeseries Collision".
+
+`Duplicate TimeSeries encountered` is the clearest indication of a timeseries
+collision. It means that two timeseries in a single request had identical
+monitored resource and metric labels.
+
+`Points must be written in order` often indicates that two different collectors
+are writing the same timeseries, since they can race to deliver the same
+metric with slightly different timestamps. If the later timestamp is delivered
+first, it triggers this error. The duplicates don't appear in the same request,
+so it doesn't trigger the `Duplicate TimeSeries encountered` error, but they do
+still collide.
+
+`points were written more frequently than the maximum sampling period` also
+often indicates that two different collectors are writing the same timeseries,
+but happens when the first timestamp is delivered first, and the later
+timestamp is delivered second. In this case, the points are in order, but are
+rejected because they are too close together.
+
+#### Root-causing Timeseries Collisions
+
+There are three main root-causes for timeseries collisions:
+
+1. Resource attributes don't distinguish applications.
+2. Resource attributes are dropped by the exporter.
+3. Metric data point attributes don't distinguish timeseries (very rare).
+
+The most common reason is (1), which means that it can be fixed by adding
+resource information. If you are running on GCP, you can use the
+`resourcedetection` processor with the `gcp` detector. If you are running on
```

Comment:
Maybe link to the processor? https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md